### PR TITLE
Updated and sort out the list of commands.

### PR DESCRIPTION
**Pull request by Danielx64**
Hello,

There's a suggestion in issue: https://github.com/Microsoft-community/Community-website/issues/16 to split out the commands into different sections. I have done this and also updated the list of commands as well.

All feedback is welcomed

Cheers

### CODE REVIEW

File: member/index.md [hunk 1]
```diff
@@ -78,21 +74,36 @@ We'd like you to bear in mind that none of our **moderators** work for nor are a
...
-| !at                     | Reminds you that mentioning someone who isn't active is not appreciated |
-| !join-insiders          | Gives you the Insiders role                              |
-| !join-insiders-slow     | Gives you the Insiders (Slow) role                       |
+
+### Staying updated Commands
+| Command                 | Function                                                 |
+| ----------------------- | -------------------------------------------------------- |
+| !join-insider-dev          | Gives you the Windows Insiders (Dev) role             |
+| !join-insider-beta     | Gives you the Windows Insiders (Beta) role                |
+| !join-insider-rp:     | Gives you the Windows Insiders (Release Preview) role      |
```
Comment (Jasius) [inline member/index.md:83]: Unnecessary symbol included here `:`

File: member/index.md [hunk 2]
```diff
@@ -78,21 +74,36 @@ We'd like you to bear in mind that none of our **moderators** work for nor are a
...
+| $nj-latestrp              | Gets the latest Release Preview Channel Insider build  |
+| $nj-latestserver        | Gets the latest Insider Server build                     |
+
+### Misc Commands
+| Command                 | Function                                                 |
+| ----------------------- | -------------------------------------------------------- |
+| !rules                  | Posts a link to the server rules                                        |
+| !rank                   | DMs you your current rank                                |
+| !levels                 | DMs you the leaderboard for the server                   |
+| !whois `<member> `      | Displays data about the mentioned member                 |
```
Comment (Jasius) [inline member/index.md:98]: Unnecessary space after <member>

Comment (Danielx64) [reply member/index.md]: Please bear with me as I try to update this on my end (still getting back into github after a good break)

**Comment by Danielx64** [comment]
Ok, I have updated it on my end and I think I pushed the 2 small changes back in?

**Pull request merged**